jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/JKubeServiceHub.java

```diff
@@ -168,4 +139,57 @@ public DebugService getDebugService() {
         return debugService.get();
     }
 
+    private void initClusterDependentServiceLazyBuilders() {
```

Comment:
Not all of these services seem to be dependent on cluster.
```suggestion
    private void initClusterAccessAndLazyBuilders() {
```